src/types.ts

```diff
@@ -739,6 +739,14 @@ export class ZodString extends ZodType<string, ZodStringDef> {
       ...errorUtil.errToObj(options?.message),
     });
   }
+  
+  numericString(message?: errorUtil.ErrMessage) {
+    return this._addCheck({
+      kind: "regex",
```

Comment:
I think it should have a kind `"numeric"` and `isNumeric()` method to align with other regex QoL checks.

Reply:
Okay. I added `"numeric"` kind and `isNumeric()`.